### PR TITLE
test(core): Add @tangleid/core test cases

### PR DESCRIPTION
**Pull request by ender503**
To test the whole @tangleid/core package, add the test case to check the
classic scenario, it will do the following steps:
 - identifier registration for the credential issuer
 - signing a credential
 - verifying the signed credential

Because we are signing the credential with the "AssertionProofPurpose",
it not only checks signature with issuer public key in DID document,
but also check whether the public key exists in the "assertionMethod"
field.

### CODE REVIEW

File: packages/core/test/core.test.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  },
+  { virtual: true }
+);
+
+describe('@tangleid/core', () => {
+  it('register identifier and generate a verifiable credential', async () => {
+    const tid = composeAPI({
+      providers: {
+        '0x1': 'http://node.deviceproof.org:14265',
+        '0x2': 'https://nodes.devnet.thetangle.org:443',
```
Comment (wusyong) [inline packages/core/test/core.test.ts]: Are we still going to use testnet node?

File: packages/core/test/utilities.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const generateSeed = (length = 81) => {
```
Comment (wusyong) [inline packages/core/test/utilities.ts]: Is it able to `generateHash` and produce any length we pass? Like maybe I want to get a tag that is 27 long tryte.

File: packages/core/test/core.test.ts (hunk 1 above)
Comment (ender503) [reply packages/core/test/core.test.ts]: Remove it. We do not recommend to use the testnet node for now.

File: packages/core/test/utilities.ts (hunk 2 above)
Comment (ender503) [reply packages/core/test/utilities.ts]: `generateHash` looks like an underlying function, and `generateSeed` should use it.
By the way, is it useful to provide the `generateSeed()` to the user of the `@tangleid/core`? 🤔  

The code here is only provided for testing.

Comment (wusyong) [reply packages/core/test/utilities.ts]: I see. If this is only for testing I think it could be a fixed one.

**Pull request merged**